harvester/harvest.py

```diff
@@ -438,9 +438,12 @@ def send_notification_emails(self, results: dict) -> None:
                     server.sendmail(SMTP_CONFIG["default_sender"], [recipient],
```

Comment:
Not part of this commit, but does function halt if it fails on one recipient? so it fails for all other?

Reply:
We can put the try within the for loop to account for that.